data_dir/scylla.yaml

```diff
@@ -118,6 +118,20 @@ backends: !mux
             ami_db_cassandra_user: 'ubuntu'
             ami_id_monitor: 'ami-79d3f06e'
             ami_monitor_user: 'centos'
+        #multiple datacenters
+        us_east_1_and_us_west_2:
```

Comment:
The config parse is done by multiplex plugin in Avocado, it passes single variant to each subtest. I didn't find a way to implement your suggestion.